ServerRoomMonitoring.Api/ServerRoomMonitoring.Api.csproj

```diff
@@ -14,6 +14,11 @@
   <ItemGroup>
     <Folder Include="Config" />
     <Folder Include="Liseners" />
+    <Folder Include="Models" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\ServerRoomMonitoring.Web\ServerRoomMonitoring.Web.csproj" />
```

Comment:
Działać to będzie oczywiście. Ale lepiej wspólne klasy wyrzycić do osobnego projektu library (taki którego nie można uruchomić). Bo teraz API będzie musiało mieć wczytany cały projekt Web, żeby mogło się uruchomić, co jest totalnie niepotrzebne. 

Ale jak mówię, w ramach studenckiego projektu, to można się tym nie przejmować.